### PR TITLE
Minimal CUDA image test (nvcc+version)

### PR DESCRIPTION
**Pull request by lugi0**
Signed-off-by: Luca Giorgi <lgiorgi@redhat.com>

### CODE REVIEW

File: tests/Tests/500__jupyterhub/minimal-cuda-test.robot [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Library          DebugLibrary
+Library          JupyterLibrary
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+*** Variables ***
+
+
+*** Test Cases ***
+Minimal CUDA test
```
[the remark this answers is not in the document]
Comment (lugi0) [reply tests/Tests/500__jupyterhub/minimal-cuda-test.robot:14]: Added in latest commit

**Comment by lugi0** [comment]
PR test:
https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/97/

**Comment by vasukulkarni** [comment]
@lugi0 if you have rerun this test on latest build can you post that link?

**Comment by lugi0** [comment]
Tested locally on 1.1 and the PR passes, here's the Jenkins link: https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/105/console
Unfortunately we are hitting many failures at the moment due to underlying product issues, your call to merge or wait for stabilization

**Comment by lugi0** [comment]
@vasukulkarni removed smoke tag

**Comment by lugi0** [comment]
Closing since won't be needed for now

**Pull request closed without merging**